mito-ai/ui-tests/tests/mitoAdvice.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { expect, test } from '@jupyterlab/galata';
```

Comment:
Just leaving this as a note that we decided we're going to move this test into the `tests` folder at the root of the monorepo. This test will then automatically run whenever we edit the `mito-ai` folder.